pkg/controller/service/servicecontroller.go

```diff
@@ -363,18 +363,15 @@ func (s *ServiceController) createExternalLoadBalancer(service *api.Service) err
 		return err
 	}
 	name := s.loadBalancerName(service)
-	if len(service.Spec.DeprecatedPublicIPs) > 0 {
-		for _, publicIP := range service.Spec.DeprecatedPublicIPs {
-			// TODO: Make this actually work for multiple IPs by using different
-			// names for each. For now, we'll just create the first and break.
-			status, err := s.balancer.CreateTCPLoadBalancer(name, s.zone.Region, net.ParseIP(publicIP),
-				ports, hostsFromNodeList(&nodes), service.Spec.SessionAffinity)
-			if err != nil {
-				return err
-			} else {
-				service.Status.LoadBalancer = *status
-			}
-			break
+	if len(service.Spec.LoadBalancerIP) > 0 {
+		// TODO: Make this actually work for multiple IPs by using different
+		// names for each. For now, we'll just create the first and break.
+		status, err := s.balancer.CreateTCPLoadBalancer(name, s.zone.Region, net.ParseIP(service.Spec.LoadBalancerIP),
```

Comment:
I was assuming the simplest feature here is that the IP is available and NOT attached to a forwarding rule (in GCE parlance).